docs/src/Tooltip.doc.js

```diff
@@ -1,165 +1,822 @@
 // @flow strict
 import React, { type Node } from 'react';
-import Example from './components/Example.js';
 import PropTable from './components/PropTable.js';
 import PageHeader from './components/PageHeader.js';
+import MainSection from './components/MainSection.js';
 
 const cards: Array<Node> = [];
 const card = (c) => cards.push(c);
 
 card(
   <PageHeader
     name="Tooltip"
-    description="The Tooltip component allows you to wrap a child with a help tooltip when hovered.
-    Tooltips are about way finding, not feature adoption, education, or promotion. They should
-    only include short descriptive text and are co-located with the element they describe."
+    description={`
+Tooltip is a floating text label that succinctly describes the function of an interactive element, typically [Icon Button](/IconButton). It’s displayed continuously as long as the user hovers over or focuses on the element.`}
   />,
 );
 
+card(
+  <MainSection name="Example" showHeading={false}>
+    <MainSection.Card
+      cardSize="lg"
+      defaultCode={`
+<Flex>
+  <Tooltip text="Align left">
+    <IconButton
+      accessibilityLabel="Align left"
+      bgColor="white"
+      icon="text-align-left"
+      iconColor="darkGray"
+      size="lg"
+    />
+  </Tooltip>
+  <Tooltip text="Align center">
+    <IconButton
+      accessibilityLabel="Align center"
+      bgColor="white"
+      icon="text-align-center"
+      iconColor="darkGray"
+      size="lg"
+    />
+  </Tooltip>
+  <Tooltip text="Align right">
+    <IconButton
+      accessibilityLabel="Align right"
+      bgColor="white"
+      icon="text-align-right"
+      iconColor="darkGray"
+      size="lg"
+    />
+  </Tooltip>
+</Flex>`}
+      showCode={false}
+    />
+  </MainSection>,
+);
+
 card(
   <PropTable
     props={[
       {
         name: 'children',
         type: 'React.Node',
+        description: `The child element, usually [Icon Button](/IconButton), that triggers Tooltip on hover or focus`,
         required: true,
-        description: 'The element to wrap with a tooltip on hover.',
       },
       {
         name: 'idealDirection',
         type: `'up' | 'right' | 'down' | 'left'`,
-        description: 'Preferred direction for the Tooltip to open',
+        description: `The text shown in Tooltip to describe its anchor element. See [localization](#Localization) to learn more.`,
```

Comment:
This seems wrong, and I think accessibilityLabel is missing?

Reply:
![image](https://user-images.githubusercontent.com/10593890/108166542-55c6f280-70c2-11eb-91b7-19fde58c3e0a.png)

Reply:
DONE re text